src/launch.js

```diff
@@ -13,10 +13,8 @@ var serviceUri = urlUtils.getUrlParameter("iss");
 var launchContextId = urlUtils.getUrlParameter("launch");
 
 // Change this to the ID of the client that you registered with the SMART on FHIR authorization server.
-var clientId = "default"; // local client
-
-
-console.log(serviceUri);
+var clientId = "7c47a01b-b7d8-41cf-a290-8ed607108e70"; // local client
```

Comment:
The hard-coding of this clientId is for ease-of-testing with Epic while we fix the issue with localStorage not being sufficient for storing clientId information.  A backend will likely be made to handle both logs and clientId/fhirServer info, but in the interim this client id will be here.  It is functionally the same thing as "default" since the app will still defer to the value saved in `localStorage`, so it won't break anything.